composer/src/main/kotlin/com/gojuno/composer/Args.kt

```diff
@@ -96,7 +97,15 @@ data class Args(
                 description = "Connected devices/emulators that will be used to run tests against. If not passed — tests will run on all connected devices/emulators. Specifying both `--device-pattern` and `--devices` will result in an error. Usage example: `--device-pattern \"somePatterns\"`.",
                 order = 10
         )
-        var devicePattern: String = ""
+        var devicePattern: String = "",
+
+        @Parameter(
+                names = arrayOf("--install-timeout"),
+                required = false,
+                description = "APK installation timeout in seconds. If not passed defaults to 120 seconds (2 minutes). Applicable to both test APK and APK under test.",
+                order = 11
+        )
+        var installTimeoutSeconds: Int = TimeUnit.MINUTES.toSeconds(2).toInt()
```

Comment:
Not sure this is how you set default value, most likely JCommander will override that anyway

Can you add a test please?